src/Collection.js

```diff
@@ -817,10 +812,6 @@ Collection.prototype.scrollSpecifications = function (scrollId, options, cb) {
 
   this.kuzzle.callbackRequired('Collection.scrollSpecifications', cb);
 
-  if (options && options.scroll) {
```

Comment:
not needed here since `scroll` option is already managed within `kuzzle.query`method